packages/oas-to-har/src/index.ts

```diff
@@ -223,9 +223,11 @@ export default function oasToHar(
   values: DataForHAR = {},
   auth: AuthForHAR = {},
   opts: oasToHarOptions = {
-    // If true, the operation URL will be rewritten and prefixed with https://try.readme.io/ in
-    // order to funnel requests through our CORS-friendly proxy.
-    proxyUrl: false,
+    // the URL of a proxy to use. Requests will be preefixed with its value;
+    // for example if you use "https://try.readme.io", a request to
+    // "https://example.com/some/api" will be sent to
+    // "https://try.readme.io/https://example.com/some/api"
+    proxyUrl: '',
```

Comment:
i think you can remove all of this, including the default value (esp since hovering over `proxyUrl` will surface that description if you use my JSDoc suggestion from above)

```suggestion
```